lib/Frontend/ArgsToFrontendOutputsConverter.cpp

```diff
@@ -133,7 +133,7 @@ OutputFilesComputer::OutputFilesComputer(
     const FrontendOptions::ActionType requestedAction,
     const llvm::opt::Arg *moduleNameArg, const StringRef suffix,
     const bool hasTextualOutput)
-    : Args(args), Diags(diags), InputsAndOutputs(inputsAndOutputs),
```

Comment:
Shouldn't the argument be dropped too? cc @davidungar

Reply:
Probably. I just wanted to quiet the unused field warning.

Reply:
Yes, please by all means drop the argument, too. (Tx @jrose-apple )